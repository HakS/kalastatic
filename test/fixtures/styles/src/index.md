---
layout: layouts/default.twig
---

## Hello World

This is a markdown file.
