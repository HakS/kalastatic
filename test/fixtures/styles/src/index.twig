---
layout: layouts/default.twig
pageTitle: Hello World!
---

<h2>{{ pageTitle }}</h2>

{% include "@kalastatic/components/custom/atoms/button.twig" with { primary: true, title: 'Hello!', active: false } only %}

<h2>Using Include with namespace</h2>
{% include "@kalastatic/components/custom/atoms/button.twig" with { title: 'Hello World!' } only %}

<h2>Button Group</h2>
{% include "@kalastatic/components/custom/molecules/button-group.twig" with { buttons: {one: {title: 'Button One'}, two: {title: 'Button Two'}} } only %}

<h2>Link</h2>
{% include "@kalastatic/components/atoms/link.twig" with { url: "http://example.com", title: "Example Link" } %}
