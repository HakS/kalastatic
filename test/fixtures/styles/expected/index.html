<!DOCTYPE html>
<html>
    <head>
        <title></title>
        <link rel="stylesheet" href="main.css">
    </head>
    <body>
        <ul id="navigation">
                </ul>

        <h1>My Webpage</h1>
        
<h2>Hello World!</h2>

  


  

	<button class="pure-button pure-button-primary">Hello!</button>


<h2>Using Include with namespace</h2>
  




	<button class="pure-button">Hello World!</button>

<h2>Button Group</h2>
  <ul>
          <li>
          




	<button class="pure-button">Button One</button>
      </li>
          <li>
          




	<button class="pure-button">Button Two</button>
      </li>
      </ul>

<h2>Link</h2>

<a href="http://example.com">Example Link</a>

    </body>
</html>
